src/main/resources/edu/cornell/kfs/fp/cu-fp-resources.properties

```diff
@@ -0,0 +1,9 @@
+error.create.accounting.document.av.amount.required=AV accounting line did not specify a debit or credit amount
```

Comment:
I think this is great to create a new FP resources properties file. Do we have a user story to move appropriate properties from application resources into this file?

Reply:
I know we had discussed moving the properties into module-specific files at some prior retro meetings, but I'm having trouble finding any follow-up user stories. We can create some new user stories for this effort if they haven't been created yet.